src/averageAges.js

```diff
@@ -14,12 +14,26 @@
  *
  * @return {number}
  */
+
+// helper fn
+function isMan(person) {
+  return person.sex === 'm';
+}
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const filteredTab = people
```

Comment:
Name this cost more descriptively
```suggestion
  const men= people
```